### PR TITLE
Replace docx with pdf for documentation download

### PR DESCRIPTION
**Pull request by maudib528**

### CODE REVIEW

**Comment by maudib528** [comment]
@delosh653 I had to resolve a conflict up there, but this should be good to go. All I did was replace docx with pdf.

File: about/docdownload.Rmd [hunk 1]
```diff
@@ -19,40 +19,40 @@ Each measure used in the MWI Tool was assessed based on its strengths, limitatio
...
 
 # Methods
-- [Facility Weights](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Facility%20Weights%20Methods.docx?raw=true){target="_blank"}
-- [Geography](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Geography%20Methods.docx?raw=true){target="_blank"}
-- [Two Step Floating Catchment Methodology](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Two%20Step%20FCA.docx?raw=true){target="_blank"}
-- [Facility Weighting](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Weighting%20Methods.docx?raw=true){target="_blank"}
\ No newline at end of file
+- [Facility Weights](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Facility%20Weights%20Methods.pdf?raw=true){target="_blank"}
+- [Geography](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Geography%20Methods.pdf?raw=true){target="_blank"}
+- [Two Step Floating Catchment Methodology](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Two%20Step%20FCA.pdf?raw=true){target="_blank"}
```
Comment (karenmjiang) [inline about/docdownload.Rmd:57]: FYI The file name for the two step FCA document has changed from the acronym to the full words :)

Comment (delosh653) [reply about/docdownload.Rmd]: Fixed!

**Review by delosh653 — APPROVED** [review]
Looks good!

**Pull request merged**